charts/kube-green/templates/deployment.yaml

```diff
@@ -82,11 +82,11 @@ spec:
           protocol: TCP
         resources:
           limits:
-            cpu: 500m
-            memory: 128Mi
+            cpu: {{ .Values.resources.limits.cpu }}
```

Comment:
Hi @zyue110026, thanks for highlighting this! I think this is a misconfiguration, since resources are moved under the `manager` field.
This container is the `kube-rbac-proxy` one, so should be set the resources below this block. So, something like

```yaml
kubeRbacProxy:
  resource:
    ...
```

I think this could be updated in this way, and the first level resources configuration could be removed

Reply:
Thanks for pointing it out. I updated the commits.

Reply:
Perfect, thank you for the update!